libraries/WiFi/src/WiFiUdp.cpp

```diff
@@ -225,7 +225,7 @@ int WiFiUDP::parsePacket(){
     rx_buffer = new cbuf(len);
     rx_buffer->write(buf, len);
   }
-  delete[] buf;
+  free(buf);
```

Comment:
Should not be used. Because it makes garbage memory area, so this function is called many time, then system cannot allocate memory in future.